### ISSUE TITLE
Support for a list of transforms rather than dict

### ISSUE THREAD

**Issue by ethanwharris** [PyTorchLightning/lightning-flash#644]
## 🚀 Feature

Enable support for just a list of transforms / a single Sequential or Compose transform to be passed to the preprocess. This could just be bound to the `pre_tensor_transform` hook by default.

**Comment by FrancescoSaverioZuppichini** [comment]
I @ethanwharris thank you for taking your time to open this issue. I think the current way is not bad but it should be documented better :)

**Comment by aniketmaurya** [comment]
Hi @ethanwharris, it will be a lot easier to have this feature. Also, we can have a method to get augmentations like below - 

```python
def get_augs(rotate_vertical:bool, rotate_horizontal:bool, shear:float):
     """do something"""
```

keras has [ImageDataGenerator](https://keras.io/api/preprocessing/image/) API which makes it easier to use for new users.

**Comment by ethanwharris** [comment]
@aniketmaurya Want to try to add it? I think we can just detect if transforms are not a dict but instead a list or callable then wrap in a dictionary. I like your suggestion of a `get_augs`, an idea we had is to make a registry for transform recipes, then we could just add a bunch of helpful defaults that can be referenced with a string passed as `*_transform`. That has the advantage of working nicely with Flash Zero. What do you think? cc @ananyahjha93 who has worked on something related to this

**Comment by aniketmaurya** [comment]
Sure @ethanwharris I will create a PR for the first task first - support transformation without dict.
